packages/daimo-api/package.json

```diff
@@ -18,9 +18,9 @@
   "repository": "https://github.com/daimo-eth/daimo.git",
   "dependencies": {
     "@coinbase/cbpay-js": "^1.8.0",
-    "@daimo/common": "*",
-    "@daimo/contract": "*",
-    "@daimo/userop": "*",
+    "@daimo/common": "0.2.10",
+    "@daimo/contract": "0.2.11",
+    "@daimo/userop": "0.1.0",
```

Comment:
just wondering, why do we need to specify the versions here if these are all in the same repo?